--- ./tensorflow/tensorflow/core/platform/env.cc.ori	2018-07-02 10:56:41.641872000 +0200
+++ ./tensorflow/tensorflow/core/platform/env.cc	2018-07-02 10:57:49.681840000 +0200
@@ -347,7 +347,7 @@
   // Has to be casted to long first, else this error appears:
   // static_cast from 'pthread_t' (aka 'pthread *') to 'int32' (aka 'int')
   // is not allowed
-  int32 tid = static_cast<int32>(static_cast<int64>(pthread_self()));
+  int32 tid = (int32)((int64)(pthread_self()));
   int32 pid = static_cast<int32>(getpid());
 #elif defined(PLATFORM_WINDOWS)
   int32 tid = static_cast<int32>(GetCurrentThreadId());
